src/traits/lowering-module.md

```diff
@@ -45,19 +45,18 @@ compiler will then invoke the `program_clauses_for` query on that
 item, and emit compiler errors that dump the clauses produced. These
 errors just exist for unit-testing, as we can then leverage the
 standard [ui test] mechanisms to check them. In this case, there is a
-`//~ ERROR Implemented` annotation which is intentionally minimal (it
-need only be a prefix of the error), but [the stderr file] contains
+`//~ ERROR program clause dump` annotation which is always the same for
+`#[rustc_dump_program_clauses]`, but [the stderr file] contains
 the full details:
 
 ```text
-error: Implemented(T: Foo) :- ProjectionEq(<T as std::iter::Iterator>::Item == i32), TypeOutlives(T \
-: 'static), Implemented(T: std::iter::Iterator), Implemented(T: std::marker::Sized).
-  --> $DIR/lower_impl.rs:15:1
+error: program clause dump
+  --> $DIR/lower_impl.rs:5:1
    |
-LL | #[rustc_dump_program_clauses] //~ ERROR Implemented(T: Foo) :-
+LL | #[rustc_dump_program_clauses]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-error: aborting due to previous error
+   |
+   = note: forall<T> { Implemented(T: Foo) :- ProjectionEq(<T as std::iter::Iterator>::Item == i32), TypeOutlives(T: 'static), Implemented(T: std::iter::Iterator), Implemented(T: std::marker::Sized). }
```

Comment:
There was a continuation of the following form before:
```
... TypeOutlives(T \
: 'static), ...
```

Not sure if that was intentional, but it did not render properly and produced `\` in the output.

I just let the line continue as it does in the actual code.

Reply:
I think that should be ok. Thanks :)